doc/examples/segmentation/plot_hausdorff_distance.py

```diff
@@ -69,6 +71,12 @@
 ax.add_artist(plt.Circle((15, 30), 10, color='y', fill=None))
 ax.add_artist(plt.Circle((45, 30), 10, color='y', fill=None))
 
+# Annotate the returned pair of points that are hausdorff distance apart
```

Comment:
```suggestion
# Annotate the returned pair of points that are Hausdorff distance apart
```

Reply:
I took the liberty of uppercasing all the uses of "hausdorff" (lowercase) in the comments.